scripts/run_integration_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Script to locally run the functional integration test.
+# This script assumes that integration tests are already built under ./bazel-bin directory
```

Comment:
We should test that the required build artifacts exist. If they don't, let's log output explaining what bazel targets need to be run and exit 1.